src/textual/widgets/_markdown.py

```diff
@@ -146,10 +147,14 @@ def build_from_token(self, token: Token) -> None:
         self._token = token
         style_stack: list[Style] = [Style()]
         content = Text()
+        repeating_whitespace = re.compile(r"[ \t]+")  # Space or tabs get reduced.
```

Comment:
```suggestion
        repeating_whitespace = re.compile(r"[ \t]+")  # Spaces and tabs get reduced.
```

Reply:
Yes, I'm _this_ annoying 😆

Reply:
I'd read it as it's making plural "space or tab", but I'm happy to change it.

Reply:
Ah, maybe it's a non-native speaker thing.
I thought the missing `s` was a typo: the translation would be wrong unless both nouns were in the plural and for some reason it felt weird in English too. 
The “and” suggestion was just because it sounded better...
But they were just _suggestions_!

Reply:
That's the difference between proper English and Yorkshire I guess. ;-)